web-client/angular.json

```diff
@@ -168,6 +168,7 @@
     }
   },
   "cli": {
+    "analytics": false,
```

Comment:
what's the relevance of this line on the registration page?

Reply:
Oh, post installing angular, it prompted me to share analytics with Google Analytics. I selected no. I think that added that line.  I'll just remove it.